packages/wallet/ui/src/components/Offer.jsx

```diff
@@ -12,6 +12,9 @@ import Proposal from './Proposal';
 
 import './Offer.scss';
 
+// XXX this isn't implemented yet so hide the button.
+const ALLOW_EXIT_OFFER = false;
```

Comment:
the `previewEnabled` could be used, but probably not worth passing it down. eventually it would make a good context.